CHANGELOG.md

```diff
@@ -556,6 +556,7 @@
 - Users can report the strategic area under which the GCRF allocation was made
 
 ## [unreleased]
+- collect additional budget types in the applicaiton
```

Comment:
When fixing the conflicts in the changelog, would you mind capitalising the sentence and fixing the typo? ("applicaiton" -> "application")